gates.py

```diff
@@ -37,3 +37,11 @@ def __str__(self):
                           dtype=np.complex256) / np.sqrt(2))
 Identity = Gate('Identity',
                np.matrix([[1.0, 0.0], [0.0, 1.0]], dtype=np.complex256))
+
+# TODO: Verify this is true
+ToZero = Gate('ZeroProjection',
+               np.matrix([[1.0, 1.0], [0.0, 0.0]], dtype=np.complex256))
```

Comment:
True, that make sense but the normalization is lost, so we'll need to normalize again after applying the operator in the last step.